shotover-proxy/tests/helpers/mod.rs

```diff
@@ -139,6 +141,32 @@ impl ShotoverManager {
         .unwrap()
     }
 
+    #[allow(unused)]
+    pub fn cassandra_connection_tls(
+        &self,
+        contact_points: &str,
+        port: u16,
+        tls_config: TlsConfig,
+        username: &str,
+        password: &str,
+    ) -> Session {
+        let ca_cert = read_to_string(tls_config.certificate_path).unwrap();
+        let mut ssl = Ssl::default();
+        Ssl::add_trusted_cert(&mut ssl, &ca_cert).unwrap();
+
+        for contact_point in contact_points.split(',') {
+            crate::helpers::wait_for_socket_to_open(contact_point, port);
+        }
+
+        let mut cluster = Cluster::default();
+        cluster.set_credentials(username, password).unwrap();
```

Comment:
why are we introducing username and password as well?
Is it tied to TLS somehow?

Reply:
The docker-image has it enabled by default, and I thought to leave it on so we can test that as well.

Reply:
Hmmm alright, sounds reasonable